tests/functional/spec/features/section_summary/section_summary.spec.js

```diff
@@ -164,4 +175,21 @@ describe("Section Summary", () => {
       expect($(HouseholdDetailsSummaryPage.heading()).getText()).to.contain("Household Summary - Semi-detached");
     });
   });
+  describe("Given I start a questionnaire that has a summary with no change links", () => {
+    it("When I then navigate to that summary, then the correct headers should be set", () => {
```

Comment:
This could be more specific in the `then`, something like `change answer should not be displayed`?